scripts/update_teams.py

```diff
@@ -100,7 +105,7 @@ def create_team(org, name, description, repos):
     team._requester.requestJsonAndCheck("PUT", url, input={"permission": "push"})
 
     current_members = team.get_members()
-    member_handles = set([member.login for member in current_members])
+    member_handles = set([member.login.lower() for member in current_members])
```

Comment:
One test is to re-run with you removed from the pandoc team to see if you are re-added. Happy for me to try?